AmericanExpress/build.gradle

```diff
@@ -5,7 +5,8 @@ plugins {
 }
 
 android {
-    compileSdkVersion rootProject.compileSdkVersion
+    namespace "com.braintreepayments.api.americanexpress"
```

Comment:
AGP 8 requires moving the namespace out of `AndroidManifest` into `build.gradle`